queries.sql

```diff
@@ -7,4 +7,59 @@ SELECT date_of_birth from animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name and escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered IS 'true';
 SELECT * from animals WHERE name != 'Gabumon';
-SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Update table
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
```

Comment:
- [ ] You are to select species from the animals table and not all the columns as you have done in line 16. Kindly fix this.
`SELECT species from animals;`

Reply:
Thank you @t-yanick , I will make the change.